### ISSUE TITLE
Better keyboard accessibility on paginated menu

### ISSUE THREAD

**Issue by alasdairhurst** [ericgio/react-bootstrap-typeahead#309]
Currently, the arrow keys can be used to loop through selections in the menu, but when the menu is paginated and shows the "show more" option, you can never reach that option with the keyboard to select it. It just loops back to the input field.

Supposedly the fix should be to count the "show more" button as a menu item when looping through the list if it is visible.

**Comment by ericgio** [comment]
This is being fixed in v3.0.0

**Comment by ericgio** [comment]
Added in v3.0.0
